pkg/cmd/helm/release/release.go

```diff
@@ -128,12 +129,14 @@ func NewCmdHelmRelease() (*cobra.Command, *Options) {
 	cmd.Flags().StringVarP(&o.VersionFile, "version-file", "", "VERSION", "the file to load the version from if not specified directly or via a $VERSION environment variable")
 	cmd.Flags().StringVarP(&o.Namespace, "namespace", "", "", "the namespace to look for the dev Environment. Defaults to the current namespace")
 	cmd.Flags().StringVarP(&o.GithubPagesBranch, "repository-branch", "", "gh-pages", "the branch used if using GitHub Pages for the helm chart")
+	cmd.Flags().StringVarP(&o.RegistryConfigFile, "registry-config", "", "/tekton/creds-secrets/tekton-container-registry-auth/.dockerconfigjson", "the path to the registry config for OCI login")
```

Comment:
Remember to run make docs to update the documentation.

Reply:
What is the default location that helm registry login looks at for registry config?

Reply:
Default location is documented here https://helm.sh/docs/helm/helm_registry_login/  it is `~/.config/helm/registry/config.json`

Reply:
I made docs, It didn't quite work but running the steps from the Makefile manually sort of did.